ParameterTrendPercentages/ParameterTrendPercentages_1/ParameterTrendPercentages_1.cs

```diff
@@ -64,6 +63,7 @@ public class CSVDataSource : IGQIOnInit, IGQIDataSource, IGQIInputArguments, IGQ
 {
     private static readonly GQIStringColumn _keyColumn = new GQIStringColumn("Key");
     private static readonly GQIDoubleColumn _percentageColumn = new GQIDoubleColumn("Percentage");
+    private static readonly GQIDoubleColumn _absoluteColumn = new GQIDoubleColumn("Absolute");
 
     private static readonly GQIStringArgument _paramIdArg = new GQIStringArgument("Parameter Id") { IsRequired = true };
```

Comment:
After having looked through the entire code I believe your start and end arguments are required.